osu.Game/Screens/Edit/Compose/Components/ComposeBlueprintContainer.cs

```diff
@@ -186,6 +207,39 @@ protected virtual IEnumerable<TernaryButton> CreateTernaryButtons()
                 yield return new TernaryButton(kvp.Value, kvp.Key.Replace("hit", string.Empty).Titleize(), () => getIconForSample(kvp.Key));
         }
 
+        private IEnumerable<TernaryButton> createSampleBankTernaryButtons()
+        {
+            foreach (var kvp in SelectionHandler.SelectionBankStates)
+                yield return new TernaryButton(kvp.Value, kvp.Key.Titleize(), () => getIconForBank(kvp.Key));
+        }
+
+        private Drawable getIconForBank(string sampleName)
+        {
+            return new Container
+            {
+                Size = new Vector2(30, 20),
+                Children = new Drawable[]
+                {
+                    new SpriteIcon
+                    {
+                        Size = new Vector2(8),
+                        Anchor = Anchor.CentreLeft,
+                        Origin = Anchor.CentreLeft,
+                        Icon = FontAwesome.Solid.VolumeOff
+                    },
+                    new OsuSpriteText
+                    {
+                        Anchor = Anchor.CentreLeft,
+                        Origin = Anchor.CentreLeft,
+                        X = 10,
+                        Y = -1,
+                        Font = OsuFont.Default.With(weight: FontWeight.Bold, size: 20),
+                        Text = $"{char.ToUpper(sampleName.First())}"
```

Comment:
```suggestion
                        Text = $"{char.ToUpperInvariant(sampleName.First())}"
```

A little safer. Guess I didn't ban this one back when...